tests/src/com.hazelcast/hazelcast/5.2.1/build.gradle

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright and related rights waived via CC0
+ *
+ * You should have received a copy of the CC0 legalcode along with this
+ * work. If not, see <http://creativecommons.org/publicdomain/zero/1.0/>.
+ */
+
+import org.graalvm.internal.tck.TestUtils
+
+plugins {
+    id "org.graalvm.internal.tck"
+}
+
+String libraryVersion = TestUtils.testedLibraryVersion
+
+dependencies {
+    testImplementation "com.hazelcast:hazelcast:$libraryVersion"
+    testImplementation 'org.assertj:assertj-core:3.22.0'
+    testImplementation 'javax.cache:cache-api:1.1.1'
+}
+
+test {
+    def strings = Arrays.asList("--add-modules", "java.se", "--add-exports", "java.base/jdk.internal.ref=ALL-UNNAMED",
```

Comment:
Why is this necessary?

Reply:
I don't know. If the test still passes, then you can remove it. I guess :)

Reply:
- Finish.
- ![image](https://user-images.githubusercontent.com/20187731/204861561-84b336a3-81b1-45e3-a9e5-7c8109f623ca.png)